test/integration_test/kubevirt_test.go

```diff
@@ -27,8 +27,8 @@ var templatePVCSpecs = map[string]string{
 const (
 	importerPodPrefix            = "importer"
 	importerPodStartTimeout      = 2 * time.Minute
-	importerPodCompletionTimeout = 20 * time.Minute
-	importerPodRetryInterval     = 10 * time.Second
+	importerPodCompletionTimeout = 30 * time.Minute
```

Comment:
we can increase it to even higher value of 60 min. In general, the timeout value should be such that there is absolutely no point in waiting beyond that time.

Reply:
Fixed.